### ISSUE TITLE
Resin despawns after new resin appears near it

### ISSUE THREAD

**Issue by Tostah** [Manello/hla_mod_XenThug#6]
Upon killing a headcrab, they will drop the normal amount of Resin. After killing another headcrab (or anything) in a position close to where the first headcrab died, the Resin from the first headcrab will despawn. 

Only happens ~50% of the time

**Comment by wytyang00** [comment]
In my case, the disappeared resins teleported to the location where the new resin appeared.

However, since I haven't seen more than 2 resins appearing at the same spot, I think all the other resins nearby might have actually despawned...

**Comment by Manello** [comment]
Yep, these two bugs are known, altough resin never really despawn, it just changes its location
